test/create-snowpack-app/create-snowpack-app.test.js

```diff
@@ -147,8 +147,7 @@ describe('create-snowpack-app', () => {
 
       // If tests didn't pass, output some relevant info into the test logs.
       if (exitCode !== 0) {
-        console.error('STDOUT', stdout);
-        console.error('STDERR', stderr);
+        console.error({msg: `FAILED TEMPLATE: ${template}`, stdout, stderr});
```

Comment:
One thing I recently remembered: with Jest the `console.*` logging may not happen linearly; i.e. it’s possible to get `INFO A`, `ERROR B`, `INFO B`, `ERROR A`. I think currently our tests just happen to log in the same order every time so this isn‘t an observed issue, but just for sanity I do like to group logs into one object for this reason.